app/src/main/java/org/simple/clinic/patient/businessid/Identifier.kt

```diff
@@ -39,6 +40,7 @@ data class Identifier(
       }
       BangladeshNationalId -> value
       EthiopiaMedicalRecordNumber -> value
+      IndiaNationalHealthId -> value
```

Comment:
NHID should have spaces when displaying similar to BP passport, can you check with the design team regarding the spaces?

Reply:
We're waiting for the design team to confirm this.